src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,9 +84,46 @@ public RobotContainer() {
 	 * joysticks}.
 	 */
 	private void configureBindings() {
-		swerve.setDefaultCommand(new DefaultDrive());
+		SWERVE.setDefaultCommand(new DefaultDrive());
 
-		driveController.a().onTrue(swerve.printOffsets());
-		driveController.x().onTrue(new InstantCommand(() -> imu.zeroYaw()));
+		InputManager.getInstance().init(
+				InputManager.Button.A_Button1, SWERVE.printOffsets(),
+				InputManager.Button.X_Button3, new InstantCommand(() -> PIGEON.zeroYaw()),
+				InputManager.Button.B_Button2, new InstantCommand(POSE_ESTIMATOR::resetOdometry));
+	}
+
+	public void periodic() {
+		POSE_ESTIMATOR.periodic();
```

Comment:
I don't think code should be calling periodic methods these should be run by the scheduler.

Reply:
So I don't need to call them for them to run? Will the PoseEstimator periodic function run automatically?

Reply:
That's my understanding. https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html#periodic()